client/question/services/CurrentThemeNameService.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+/**
+ * @fileoverview A service that stores the user's light/dark mode preference.
+ */
+
+tie.constant('THEME_NAME_LIGHT', 'Light Theme');
+tie.constant('THEME_NAME_DARK', 'Dark Theme');
+
+tie.factory('CurrentThemeNameService', [
```

Comment:
Why not just ThemeService or ThemeNameService? CurrentThemeNameService is ... very specific. Seems like calling it ThemeService or TieThemeService would allow us to offload more theme options to this service in the future, should we expand it out.

Reply:
Done.